### ISSUE TITLE
"SPCM" typo (should be "SCPM")

### ISSUE THREAD

**Issue by Rudxain** [0x192/Universal-Android-Debloater#533]
**Your phone**: SM-A315G

**Packages documentation to update:**
```
com.samsung.android.sm.policy
```
### com.samsung.android.sm.policy

The display-name in the docs says "SPCM", but (in my device) it's SCPM

**Comment by Rudxain** [comment]
Wait, I found more info about the package! It seems that package is the "replacement" of ["Samsung Security Policy Update"](https://play.google.com/store/apps/details?id=com.policydm), despite having different IDs.

Perhaps "SCPM" stands for "SeCurity Policy Manager"?

While Googling, I found [this](https://samsung.com/levant/support/mobile-devices/what-is-security-enhancement-for-android)

**Comment by AnonymousWP** [comment]
So if I understand it correctly: `com.samsung.android.sm.policy` is the name for both packages or is it SPCM? Is it in fact the same (so rather a typo), or are they two different packages/apps?

**Comment by AnonymousWP** [comment]
Check the PR I just submitted, hopefully it's correct like this.

**Comment by Rudxain** [comment]
> `com.samsung.android.sm.policy` is the name for both packages [...]?

No. The legacy "Security Policy Update" is its own thing (I guess?). On my new device, that updater is integrated directly into the main OTA software updater, rather than an individual app.

> [...] are they two different packages/apps?

Depends. They definitely have different IDs. But they are related. Perhaps, SCPM is not a replacement, but a "complement". (This is mostly speculation. "take it with a grain of salt")

**Comment by Rudxain** [comment]
I forgot to mention, the app icon is **cloud-shaped**. This implies the acronym stands for **"Samsung Cloud Policy Manager"**

**Comment by AnonymousWP** [comment]
Okay, so what do you expect me to change?

**Comment by Rudxain** [comment]
@AnonymousWP I think the PR is ok as it is. But it could be more informative if we include "On some devices, the app icon is cloud-shaped, which implies it's related to S Cloud"

**Comment by AnonymousWP** [comment]
> On some devices, the app icon is cloud-shaped, which implies it's related to S Cloud

Added, see PR. Is it ok?

**Comment by Rudxain** [comment]
> Is it ok?

Yes 👍

**Comment by Rudxain** [comment]
I accidentally stumbled upon the meaning! Look at the "full" permissions of `com.samsung.android.lool`:
![](https://github.com/0x192/universal-android-debloater/assets/76864299/097e809f-571b-4517-819f-a22d50fb580b)

It stands for **"Samsung Cloud Platform Manager!".** Description:
!["Allow the app to read data from the Samsung Cloud Platform Manager in order to authenticate your phone and configure your apps and services." OK button](https://github.com/0x192/universal-android-debloater/assets/76864299/10b935c5-dc5d-444b-9ff4-20e69740bb28)

**Comment by AnonymousWP** [comment]
Added a commit. What do you think of it?

**Comment by Rudxain** [comment]
> Added a commit. What do you think of it?

It's good, but the "implies it's related to S Cloud" has become redundant now. The icon shape may be redundant too, but I think it's better to keep it, just in case S changes the icon again

**Comment by AnonymousWP** [comment]
Fixed in a new commit. Better now?

**Comment by Rudxain** [comment]
LGTM.

BTW, I've uninstalled it for almost a week, and haven't noticed a change in battery life. So we should (probably) remove the "kills BG apps" and "battery impact" parts

**Comment by AnonymousWP** [comment]
> LGTM.
> 
> BTW, I've uninstalled it for almost a week, and haven't noticed a change in battery life. So we should (probably) remove the "kills BG apps" and "battery impact" parts

Fixed in a new commit.
